src/pages/singlePlaylist/SinglePlaylist.jsx

```diff
@@ -0,0 +1,52 @@
+import { Grid } from "@mui/material";
+import React from "react";
+import { useParams } from "react-router-dom";
+import { EmptyResult, PageHeader, PrivateCard } from "../../components";
+import { useAuth, useVideo } from "../../context";
+
+export const SinglePlaylist = () => {
+  const {
+    videoState: { playlists },
+    videoDispatch,
+    deleteVideoFromPlaylist,
+  } = useVideo();
+
+  const {
+    authState: { token },
+  } = useAuth();
+
+  const { slug } = useParams();
+
+  const currentPlaylist = playlists?.find((playlist) => playlist._id === slug);
```

Comment:
There is a route for single playlist too. You should use that.

Reply:
Yes that is right, will make the necessary changes